### PR TITLE
Funding card values

### PR DESCRIPTION
**Pull request by hpmaxi**

### CODE REVIEW

File: app/src/util/tools.ts [hunk 1]
```diff
@@ -175,3 +175,26 @@ export const getIndexSets = (outcomesCount: number) => {
   const range = (length: number) => [...Array(length)].map((x, i) => i)
   return range(outcomesCount).map(x => 1 << x)
 }
+
+export const calcPoolTokens = (
```
Comment (fvictorio) [inline app/src/util/tools.ts:179]: Tests.

File: app/src/util/tools.ts [hunk 2]
```diff
@@ -175,3 +175,26 @@ export const getIndexSets = (outcomesCount: number) => {
...
+  if (poolShareSupply.gt(0)) {
+    const poolWeight = holdingsBN.reduce((max: BigNumber, h: BigNumber) => (h.gt(max) ? h : max))
+
+    return addedFunds.mul(poolShareSupply).div(poolWeight)
+  } else {
+    return addedFunds
+  }
+}
+
+export const calcDepositedTokens = (
```
Comment (fvictorio) [inline app/src/util/tools.ts:193]: Tests.

Comment (hpmaxi) [reply app/src/util/tools.ts:193]: I added some tests, but I suppose there are some edge cases that I'm not addressing
